app.test.js

```diff
@@ -0,0 +1,81 @@
+const app = require("./app.js");
+const fs = require("fs/promises");
+const request = require("supertest");
+const seed = require("./db/seeds/seed.js");
+const connection = require("./db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("./db/data/test-data/index.js");
+const express = require("express");
+const { parse } = require("path");
+app.use(express.json());
+
+afterAll(() => {
+  connection.end();
+});
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
+  res.status(500).send({ msg: "Code 500 - server error!" });
+});
+
+describe("/api", () => {
+  test("POST request - status 201 posts the endpoints", () => {
```

Comment:
I believe you may have refactored this, but not a post request or a 201 here for fetching data.